front/app/components/IdeasMap/IdeaMapCard.tsx

```diff
@@ -228,19 +206,6 @@ const IdeaMapCard = memo<Props>(
           role="button"
           tabIndex={0}
         >
-          {smallerThanMaxTablet && (
-            <CloseButtonWrapper>
-              <CloseButton
-                width="26px"
-                height="26px"
-                padding="0px"
-                buttonStyle="secondary"
-                icon="close"
-                iconSize="12px"
-                onClick={handleCloseButtonClick}
-              />
-            </CloseButtonWrapper>
-          )}
```

Comment:
Was never shown because the IdeaMapCard is only rendered when !smallerThanMaxTablet 🙃